src/main/java/mekanism/common/util/MinerUtils.java

```diff
@@ -18,7 +19,7 @@
     private MinerUtils() {
     }
 
-    public static List<ItemStack> getDrops(ServerWorld world, BlockPos pos, boolean silk, BlockPos minerPosition) {
+    public static List<ItemStack> getDrops(ServerWorld world, BlockPos pos, boolean silk, BlockPos minerPosition, UUID minerOwner) {
```

Comment:
Part of me wonders if it is at the point that it may make more sense to just make this be a private method in `TileEntityDigitalMiner` rather than having to keep adding more params to this. It originally was in its own class because of how much extra logic was needed for figuring out the drops but with loot contexts it doesn't seem really necessary anymore.